manifests/init.pp

```diff
@@ -398,6 +393,13 @@
     }
   }
 
+  if $pulp or $pulp_master {
+    class { 'crane':
```

Comment:
Heh, it does, sort of. How about -- https://github.com/Katello/puppet-capsule/blob/master/manifests/init.pp#L259